extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -405,6 +441,22 @@ export class AmpAnalytics extends AMP.BaseElement {
     }
     const typeConfig = this.predefinedConfig_[type] || {};
 
+    // transport.iframe is only allowed to be specified in typeConfig, not
+    // the others. Allowed when running locally for testing purposes.
+    [defaultConfig, inlineConfig, this.remoteConfig_].forEach(config => {
```

Comment:
you can leave this out, we have a plan to make the whole transport non-overridable: #7096